design.md

```diff
@@ -0,0 +1,106 @@
+# oras multi-arch feature design, Version July 15
+
+## PoC
+
+https://github.com/wangxiaoxuan273/oras/pull/1
+
+Only implemented `oras manifest index create`.
+
+## Design
```

Comment:
In addition to OCI image index, should ORAS image index be compatible with docker manifest list?

Reply:
We should only create oci image index. If the user gives a docker manifest list to the update command, we should create a updated image index to the user.